File: 프로그래머스/3/12987. 숫자 게임/숫자 게임.java
import java.util.*;

class Solution {
    public int solution(int[] A, int[] B) {
        
        Arrays.sort(A);
        Arrays.sort(B);
        
        int nowA = 0;
        int answer = 0;
        
        for(int i=0; i<B.length; i++){
            if(B[i] > A[nowA]){
                answer++;
                nowA++;
            }
        }
            
        
        return answer;
    }
}

File: 프로그래머스/3/12987. 숫자 게임/숫자 게임.js
function solution(A, B) {
    
    A.sort((a,b) => a-b)
    B.sort((a,b) => a-b)
    
    var aIndex = 0;
    var answer = 0;
    
    for(let i=0; i<B.length; i++){
        if(B[i] > A[aIndex]){
            aIndex++;
            answer++;
        }
    }   
    
    return answer;
}